|diff --hintful a/luhn.test.js b/luhn.test.js
|index 863ee9f..2763cf7 100644
|--- a/luhn.test.js
|+++ b/luhn.test.js
|@@ -1,4 +1,3 @@
|-const assert = require('assert');
| const luhn = require('./luhn.js');
| 
| const testCases = [
|@@ -13,5 (17) +12,7 @@ const testCases = [
| ];$
| $
| for(const {str, result} of testCases){$
|+    test(JSON.stringify({str, result}), ()=>{$
|     \
|+    \
|-assert\
|+expect\
| (\
| luhn.check(str)\
|-==\
|+).toBe(\
| result\
|-, `Failed test:${JSON.stringify({str, result})}`)$
|+);$
|+    })$
| }$
diff --git a/luhn.test.js b/luhn.test.js
index 863ee9f..2763cf7 100644
--- a/luhn.test.js
+++ b/luhn.test.js
@@ -1,4 +1,3 @@
-const assert = require('assert');
 const luhn = require('./luhn.js');
 
 const testCases = [
@@ -13,5 +12,7 @@ const testCases = [
 ];
 
 for(const {str, result} of testCases){
-    assert(luhn.check(str)==result, `Failed test:${JSON.stringify({str, result})}`)
+    test(JSON.stringify({str, result}), ()=>{
+        expect(luhn.check(str)).toBe(result);
+    })
 }
diff --git a/package.json b/package.json
index e468e70..85980a5 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "description": "Luhn test library",
   "scripts": {
-    "test": "node luhn.test.js"
+    "test": "npx jest"
+  },
+  "devDependencies": {
+    "jest": "27.0.6"
   }
 }
